mangaki/mangaki/settings.py

```diff
@@ -7,10 +7,13 @@
 https://docs.djangoproject.com/en/1.9/ref/settings/
 """
 
-import os
 import configparser
 import json
+import os
+
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+FIXTURE_DIR = os.path.join(os.path.dirname(BASE_DIR), 'fixtures')
+TEST_FIXTURE_DIR = os.path.join(FIXTURE_DIR, 'tests')
```

Comment:
Yes, to me, `fixtures` were more "initial data for something (models / tests / …)", in Django, it may be reserved to models, but for testing, you also have whole libraries named `fixtures`.
Anyway, I agree with this.